src/Sushi.php

```diff
@@ -137,4 +137,28 @@ public function usesTimestamps()
             ? parent::usesTimestamps()
             : false;
     }
+
+    public function invalidateCache()
+    {
+        touch($this->cacheDirectory().'/'.$this->getCacheFileName(), filemtime($this->getModelPath()) - 10000);
+
+        return $this;
+    }
+
+    public function getCacheDirectory()
+    {
+        return $this->cacheDirectory ?? config('sushi.cache-path', storage_path('framework/cache'));
```

Comment:
According to config/sushi.php, should be 'sushi.cache-directory' instead of 'sushi.cache-path'